src/docs/arc42/adrs/ADR002-Technology-Stack.adoc

```diff
@@ -0,0 +1,91 @@
+= ADR-002: Technology Stack Selection - Python 3.8+ and MCP Protocol
+
+|===
+| Datum:    | 2025-01-13
+| Autoren:  | AsciiDoc MCP Development Team
+| Status:   | Accepted
+|===
+
+== Problem
+
+The AsciiDoc MCP Server requires a technology stack that balances developer productivity, ecosystem maturity, performance, and compatibility with the Model Context Protocol specification.
+
+== Alternatives Considered
+
+[options="header",cols="2,2,2,2"]
+|===
+|Technology|Pros|Cons|Score
+|**Python 3.8+**|• Mature AsciiDoc ecosystem +
+• Strong async support +
+• Cross-platform compatibility +
+• Rich standard library|• Runtime dependency +
+• Potential performance overhead|**+2**
+
+|**Node.js/TypeScript**|• Native JSON-RPC support +
+• Strong async model +
+• Fast startup times|• Limited AsciiDoc tooling +
+• Additional build complexity|**0**
+
+|**Go**|• Single binary deployment +
+• Excellent performance +
+• Strong concurrency|• Minimal AsciiDoc ecosystem +
+• More complex for document processing|**-1**
+
+|**Rust**|• Maximum performance +
+• Memory safety +
+• Growing ecosystem|• Steep learning curve +
+• Limited AsciiDoc libraries|**-2**
+|===
+
+== Decision
+
+**Adopt Python 3.8+ as the runtime with MCP 1.0+ protocol compliance.**
+
+**Technology Components:**
```

Comment:
user ".title" headlines instead of bold text
add an empty line before a list
Please fix all tables in the architecture accordingly

add these rules to the custom instructions file